front/pages/api/w/[wId]/labs/transcripts/index.ts

```diff
@@ -56,17 +58,12 @@ async function handler(
   }
 
   switch (req.method) {
-    // List.
-    // TODO: This should be a proper list operation.
     case "GET":
       const transcriptsConfigurationGet =
-        await LabsTranscriptsConfigurationResource.findByUserWorkspaceAndProvider(
-          {
-            auth,
-            userId,
-            provider: req.query.provider as LabsTranscriptsProviderType,
-          }
-        );
+        await LabsTranscriptsConfigurationResource.findByUserWorkspace({
+          auth,
```

Comment:
name of function and arguments do not align. I don't understand what this is doing.

Reply:
Finding a Labs transcripts resource by User and Workspace. The workspace is encapsulated in the passed `Authenticator`

Reply:
And not the user?

Reply:
Yeah... the user as well 🥹 Sry, it was from my early days playing with Authenticator, went through the first PR a while ago.
Changed, only auth is passed now.

Reply:
👍